clients/admin-ui/src/features/config-wizard/OrganizationInfoForm.tsx

```diff
@@ -132,21 +131,12 @@ const OrganizationInfoForm = () => {
     >
       <Stack spacing={10}>
         <Heading as="h3" size="lg">
-          Tell us about your business
+          Create your Organization
         </Heading>
         <div>
           Provide your organization information. This information is used to
-          configure your organization in Fidesctl for{" "}
-          <Tooltip
-            fontSize="md"
-            label="Wondering what a data map is? No problem, we've got your covered with this quick overview here"
-            placement="right"
-          >
-            <Text display="inline" color="complimentary.500">
-              data map
-            </Text>
-          </Tooltip>{" "}
-          reporting purposes.
+          configure your organization in Fidesctl for data map reporting
```

Comment:
Figma still says "Fidestctl" but since we're here maybe we should drop the "ctl"?

Reply:
Makes sense! Thanks 😁